# Note: Name of file should match name of robot in URDF

# Robot Execution Speed
main_velocity_scaling_factor: 0.5 # during planning
approach_velocity_scaling_factor: 0.25
lift_velocity_scaling_factor: 0.5
retreat_velocity_scaling_factor: 0.8
calibration_velocity_scaling_factor: 0.05

# Sleep timers
wait_before_grasp: 0.1
wait_after_grasp: 0.1

# Poses
start_pose: both_neutral
dropoff_pose: goal_bin_pose

# Semantics
dual_arm: 1
right_hand_name: right_hand
left_hand_name: left_hand
right_arm_name: right_arm
left_arm_name: left_arm
both_arms_name: both_arms

# Calibration
shelf_distance_from_robot: 1.0

# Topics
joint_state_topic: /robot/joint_states